.github/workflows/ci-build-publish-main.yml

```diff
@@ -44,3 +44,25 @@ jobs:
           $DOCKER_TAG_SHA
       env:
         COSIGN_EXPERIMENTAL: "true"
+  
+  app_deployment:
+    env:
+      cd_workflow_dispatch_file_name: cd-app-deployment.yml
+      infrastructure_repo_path: buerokratt/Infrastructure
+      app_name: mock-classifier
+    runs-on:
+      ubuntu-latest
+    defaults:
+      run:
+        shell : bash
+    needs :
+      PackageDeploy
+    steps:
+      - name: Initiate k8s app deployment
+        run: |
+          gh workflow run $cd_workflow_dispatch_file_name -f app_name=$app_name -f image_tag=${GITHUB_SHA} --repo $infrastructure_repo_path --ref main
+        env:
+          GITHUB_TOKEN: ${{ secrets.MULTI_REPO_WORKFLOW_ACCESS_TOKEN }}
```

Comment:
Is it a good idea? Usually envs and secrets starting with `GITHUB_` are special and managed by GitHub.

Reply:
[https://docs.github.com/en/actions/security-guides/encrypted-secrets#naming-your-secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets#naming-your-secrets) There is written, that `GITHUB_` prefix is not allowed for secrets. I'm guessing it goes also for env values.